scheduler/scheduler-server/src/test/java/functionaltests/StandardTestSuite.java

```diff
@@ -205,7 +206,7 @@
                       JettyStarterTest.class, TestXMLTransformer.class, TagTest.class,
 
                       // RM proxy tests
-                      TestRMProxy.class, TestRMProxyRebind.class,
+                      TestRMProxy.class, TestRMProxyRebind.class, TestRMReconnectionWhileRunning.class,
```

Comment:
This test was missing from the standard test suite? Or is it a new test (in this case it does not appear in the PR)?